src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -118,7 +131,8 @@ interface INodeLibraryDependencyInfo
         /// <summary>
         /// Indicates whether this dependency is loaded in the current session
         /// </summary>
-        bool IsLoaded { get; set; }
+        [Obsolete("This property is obsolete", false)]
+        bool IsLoaded { get; }
```

Comment:
we cannot remove the setter - we'll have to leave it - perhaps you can implement it to set the state to Loaded?

Reply:
Added back the setter and I am setting this when the state is modified. I have to do this because we wanted to check this IsLoaded property in the tests.